### ISSUE TITLE
[BUG] error with document Exporter and importer

### ISSUE THREAD

**Issue by damianarnold1993** [paperless-ngx/paperless-ngx#3618]
### Description

I've had Paperless-NGX on a Raspberry Pi using Docker-Compose for 2 years. Now I improved my homelab and implemented a Proxmox cluster. Now I want to migrate the paperless NGX.

Old setup: Raspberry Pi 4 2GB with Rasbian (32bit) and Docker. Data storage using AutoFS mounted from my Synology RackStation (backups every 24 hours)

New setup: Proxmox with a VM with Debian11 (64bit) and also Docker. the cluster offers high availability.

1. Attempt for seamless migration: wrong bitrate of POSTGRESQL database --> error
2.Try with LXC Container: AutoFS is not supported...
3.Try with Exporter/Importer: no manifest.json is recognized on the new Debian. I removed the mailrules according to BUG #816, still unsuccessful.
3.1 I tried with create Superuser and without, both paint the same result

I changed all paths to the new directory in the configuration.yaml, in case anyone doubts that ;)

Anyone an idea?

### Steps to reproduce

1. pi@raspi4:/home/nfs/Daten/Paperless-ngx/configure $ sudo docker-compose exec webserver document_exporter ../export/
![image](https://github.com/paperless-ngx/paperless-ngx/assets/76855878/b66c83e6-9346-4142-b897-9e4e2be68980)

2. copy the hole Folder to the new Directory

3. root@PaperlessNGX:/home/nfs/Daten/Paperless-ngx/NEU/configure $ sudo docker-compose exec webserver document_importer ../export/
CommandError: That directory doesn't appear to contain an manifest.json file.
![image](https://github.com/paperless-ngx/paperless-ngx/assets/76855878/41fd869b-8f95-4f0b-aadb-4e81727af5d1)

### Webserver logs

```bash
according steps to reporduce
```

### Paperless-ngx version

1.16.1

### Host OS

Raspberry Pi and Proxmox VM

### Installation method

Docker - official image

### Browser

Chrome

**Comment by stumpylog** [comment]
This seems likely to be an issue with your folders.  It's very easy to get confused about host vs image or vm folders.  

Did you mount the copied folder to where ../export points? Is there a manifest in there?  Are the documents in there?

**Comment by damianarnold1993** [comment]
yes the folder contains 1006 documents, many WEBP files and a searched Manifest.json and a version.json
in total, the Pi exports 3184 files
I can see the files on the new VM via the command line

**Comment by stumpylog** [comment]
The OS is saying no `manifest.json` exists in the given directory.  It's unlikely to be doing something incorrect.  Case does matter, so if the file is actually *M*anifest.json, that would be a problem

**Comment by damianarnold1993** [comment]
sorry my fault... the manifest.json with little letter
![image](https://github.com/paperless-ngx/paperless-ngx/assets/76855878/476b9085-4af2-4b06-8832-94f23ae09511)

**Comment by stumpylog** [comment]
I'd suggest trying the import with the fully qualified path.  So to wherever you have mapped the host folder containing the files into the container or VM, to eliminate any confusion about where the relative path is pointing.

**Comment by damianarnold1993** [comment]
doesen't work. i tried some different path, no sucess. 
![image](https://github.com/paperless-ngx/paperless-ngx/assets/76855878/7aadd7e6-0d2e-4e8b-99bd-f51fc12f59b3)


when i switch to the path, there are de searced documents.
![image](https://github.com/paperless-ngx/paperless-ngx/assets/76855878/3c527e25-56f7-4084-8a8f-ae3fe8336801)


The Path is: /home/nfs/Daten/Paperless-ngx/NEU/configure/export

I tried, to store the export-folder also in the home directory... no sucess...
